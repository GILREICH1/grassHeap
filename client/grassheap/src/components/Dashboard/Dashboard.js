import Tasks from "../Tasks/Tasks/Tasks";
import React from 'react';
import Weather from "../Weather/Weather";
import './Dashboard.css'

function Dashboard(){
  return (
    <div>
      <Weather/>
      <Tasks/>)
    </div>
  )
}

export default Dashboard;
