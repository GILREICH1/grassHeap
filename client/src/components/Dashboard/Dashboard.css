.btn {
  border-radius: 5px;
  cursor: pointer;
  border-color: transparent;
  /* border-style:none */
}
