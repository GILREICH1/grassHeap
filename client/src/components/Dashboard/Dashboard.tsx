import React from 'react';
import MyPlantsList from '../MyPlants/MyPlantsList';
import Tasks from '../Tasks/Tasks/Tasks';
import Weather from '../Weather/Weather';
import styles from './Dashboard.module.scss';

function Dashboard(): JSX.Element {
  return (
    <div className={styles.dashboard}>
      <Weather />
      <Tasks />
      <MyPlantsList />
    </div>
  );
}

export default Dashboard;
