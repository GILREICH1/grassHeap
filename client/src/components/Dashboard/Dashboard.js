import Tasks from "../Tasks/Tasks/Tasks";
import React from "react";
import Weather from "../Weather/Weather";
import "./Dashboard.css";

function Dashboard() {
  return (
    <div>
      <Weather />
      <Tasks />
      {/* <img
        className="daisies"
        src="https://uploads-ssl.webflow.com/5ea82bf973106e1765b18a5f/5fd0d4198fecb12496f3bbdb_tumbs-up-4_1.gif"
        alt="swaying daisies"
      ></img> */}
    </div>
  );
}

export default Dashboard;
