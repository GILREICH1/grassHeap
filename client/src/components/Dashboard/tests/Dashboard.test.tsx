import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from '../Dashboard';

test('layout', () => {
  const tree = renderer.create(<Dashboard />).toJSON();
  expect(tree).toMatchSnapshot();
});

export {};
