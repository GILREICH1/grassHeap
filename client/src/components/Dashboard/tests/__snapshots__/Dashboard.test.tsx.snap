// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout 1`] = `
<div
  className="dashboard"
>
  <div
    className="Weather"
  >
    <h1>
      No weather found for 
       :(
      <a
        onClick={[Function]}
      >
         try again
      </a>
    </h1>
  </div>
  <div
    className="Tasks"
  >
    <button
      className="scroll-btn scroll-btn__back"
      disabled={false}
      onClick={[Function]}
    >
      <p>
        ▶
      </p>
    </button>
    <div
      className="tasks__allmonths"
    >
      <div
        className="tasks__month tasks__month--11"
      >
        <div
          className="MonthTaskBox"
        >
          <h2>
            December
             
            
          </h2>
          <div
            className="TaskList"
          >
            <div
              className="TaskList_empty"
            >
              <p>
                No tasks for your plants this month!
              </p>
            </div>
          </div>
          <form
            className="submitForm"
            onSubmit={[Function]}
          >
            <h3>
              Add New Task
            </h3>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Task
              </label>
            </div>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Crop
              </label>
            </div>
            <input
              className="submit"
              type="submit"
              value="Submit"
            />
          </form>
        </div>
      </div>
      <div
        className="tasks__month tasks__month--0"
      >
        <div
          className="MonthTaskBox"
        >
          <h2>
            January
             
            
          </h2>
          <div
            className="TaskList"
          >
            <div
              className="TaskList_empty"
            >
              <p>
                No tasks for your plants this month!
              </p>
            </div>
          </div>
          <form
            className="submitForm"
            onSubmit={[Function]}
          >
            <h3>
              Add New Task
            </h3>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Task
              </label>
            </div>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Crop
              </label>
            </div>
            <input
              className="submit"
              type="submit"
              value="Submit"
            />
          </form>
        </div>
      </div>
      <div
        className="tasks__month tasks__month--1"
      >
        <div
          className="MonthTaskBox"
        >
          <h2>
            February
             
            
          </h2>
          <div
            className="TaskList"
          >
            <div
              className="TaskList_empty"
            >
              <p>
                No tasks for your plants this month!
              </p>
            </div>
          </div>
          <form
            className="submitForm"
            onSubmit={[Function]}
          >
            <h3>
              Add New Task
            </h3>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Task
              </label>
            </div>
            <div
              className="form__group"
            >
              <input
                className="form__field"
                onChange={[Function]}
                required={true}
                value=""
              />
              <label
                className="form__label"
              >
                Crop
              </label>
            </div>
            <input
              className="submit"
              type="submit"
              value="Submit"
            />
          </form>
        </div>
      </div>
    </div>
    <button
      className="scroll-btn "
      disabled={false}
      onClick={[Function]}
    >
      <p>
        ▶
      </p>
    </button>
  </div>
</div>
`;
